plugins/explain.py

```diff
@@ -0,0 +1,87 @@
+        'fixes': 'We use bug prediction in coala which relies on the `Fixes` '
+                 'keyword in commit messages. To get good results from that '
+                 'we need to use `Fixes` for bugfix issues instead of '
+                 '`Closes`. (See also [the commit message docs]'
+                 '(https://coala.io/commit).) To change your message you '
+                 'just use `git commit --amend` and then `git push --force` '
+                 'the new commit to replace the old one.',
+        'commit message': 'To change your message you just use `git commit '
+                          '--amend` and then `git push --force` the new '
+                          "commit to replace the old one.\n\nIf you're just "
+                          'looking to fix an issue very quickly and not '
+                          'interested in contributing to coala long term, we '
+                          'can fix up the message for you - just tell us :).',
+        'rebase': 'It looks like your PR is out of date and needs a rebase.'
+                  '\n\n[This page](https://coala.io/git) may help you to get '
+                  'started on this. We also have [a quick video tutorial on '
+                  'how to rebase](https://asciinema.org/a/78683). That should '
+                  'help you understand the basics of how it works and what you'
+                  "should be doing.\n\nIf you're just looking to fix an issue "
+                  'very quickly and not interested in contributing to coala '
+                  'long term, we can fix it up for you - just tell us :).',
+        'cep': "At coala we're using [cEP's (coala Enhancement Proposals)]"
+               '(http://coala.io/cep) to define major design decisions - '
+               "they're a bit like PEP's but not quite as extensive and "
+               'obviously written with a lower case c.',
+        'gitlab': 'We are currently evaluating on if we want to use GitLab for'
+                  "code hosting. That's why some repositories are already on "
+                  'GitLab, if you want to participate in the migration '
+                  'discussion, please add information [at our GitLab wiki page]'
+                  '(https://github.com/coala/coala/wiki/GitLab).',
+        'google': 'Hey. This message was triggered because someone was too '
+                  "lazy to type this *again*. Don't take it personally. "
+                  'Please.\n\nWe all got to learn this: *use google*. Or '
+                  'duckduckgo. Anything. The search engine that earned your '
+                  'trust. You got a build error? Search for the first red '
+                  'thing and google it. You got an exception? *Read the '
+                  'message.* Search it. *Think.*\n\nKeep this in mind: *You*'
+                  'are sitting in front of the problem, not we. You will have '
+                  "a much easier time solving it. That's why you should try "
```

Comment:
Also, single quotes is enforced by `QuotesBear` and you wouldn't have to manually check the quotes now :wink: